src/commands/deploy.ts

```diff
@@ -190,6 +204,18 @@ export const deploy = async (config: DeployConfig) => {
       VersionLabel: versionLabel
     })
 
+    // if custom cf distribution, update it
+    if (customCFDistribution) {
+      await updateDistribution(cloudfrontClient, customCFDistribution, {
```

Comment:
we should do that inside stack, otherwise we will keep changing cloudfront with each deployment

Reply:
same as the above comment. I searched and there is no way to update the distribution inside the stack.
https://github.com/aws/aws-cdk/issues/12524

Reply:
can we check at least that those origins and rules were already added, so we can skip that step?

Reply:
I will create a function to check this then.